config/yamllint.yaml

```diff
@@ -0,0 +1,31 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+
+rules:
+  braces: enable
+  brackets:
+    min-spaces-inside: 1
+    max-spaces-inside: 3
+  colons: enable
+  commas: enable
+  comments: enable
+  comments-indentation: enable
+  document-end: disable
+  empty-lines: enable
+  empty-values: disable
+  hyphens: enable
+  indentation: enable
+  key-duplicates: enable
+  key-ordering: disable
```

Comment:
Please put comment as reason of disable.
Do for all

Reply:
done